src/installation/guides/chroot.md

```diff
@@ -168,6 +168,9 @@ Update the package manager and install `base-system`:
 # xbps-remove base-voidstrap
 ```
 
+If on aarch64, it will be necessary to install a kernel package separately. For
+example, the `linux` package can be installed in addition to `base-system`.
+
```

Comment:
I'm not sure this is in the right section? Shouldn't it happen before the steps to install `base-system`?

Reply:
From my experience the kernel package could be installed before, at the same time or after `base-system` without affecting the final result much. The `xbps-reconfigure -fa` step in the end should make it work regardless?

I rewrote and moved it to right after "Base Installation". I also added mention of aarch64 in "The XBPS Method".